### PR TITLE
update: cart page added

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
Created product added into cart and price details

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -312,6 +313,83 @@ i:hover {
...
+    margin:0 1rem 0.2rem 1rem;
+}
+.prod-in-cart .text-lin-through{
+    text-decoration: line-through;
+}
+.prod-in-cart .grey-text{
+    color:var(--little-dark-grey);
+}
+.prod-in-cart img {
+    width: 180px;
```
Comment (harshalini) [inline style.css:346]: Use rem instead of px

File: style.css [hunk 2]
```diff
@@ -312,6 +313,83 @@ i:hover {
...
+    align-items: center;
+}
+.quant i{
+    border:1px solid var(--little-light-grey);
+    border-radius: 50%;
+    padding:0.5rem;
+}
+.quant input{
+    width:10%;
+    margin:0.5rem;
```
Comment (harshalini) [inline style.css]: If the value is less than 10px, use can use px directly instead of rem

**Review by harshalini — CHANGES_REQUESTED** [review]

File: style.css (hunk 2 above)
Comment (Kothalkarpriya) [reply style.css]: Yes, but here it is 50px just because of it I used rem

File: style.css (hunk 1 above)
Comment (Kothalkarpriya) [reply style.css:346]: I will change it

**Review by mariyasada — APPROVED** [review]

**Review by harshalini — APPROVED** [review]

**Pull request merged**